include/volk/sse2neon.h

```diff
@@ -0,0 +1,8854 @@
+// Turn off Clang formatting, as 
```

Comment:
What is the original copyright on this file? Hopefully it's GPL/ LGPL 3+ ... assuming it's compatible, please add the SPDX line at the top

Reply:
It is MIT (https://github.com/DLTcollab/sse2neon/blob/master/LICENSE), so as far as I'm aware, it is compatible.

Reply:
Yes MIT is forward compatible to GPL ... so just add in the SPDX lines & we're good to go

Reply:
Done